cypress/support/pages/backoffice/company-role/list/company-role-list-repository.ts

```diff
@@ -0,0 +1,12 @@
+import { autoWired } from '@utils';
+import { injectable } from 'inversify';
+
+@injectable()
+@autoWired
+export class CompanyRoleListRepository {
+  getFirstTableRow = (): Cypress.Chainable => cy.get('tbody > :nth-child(1):visible');
+  getSearchSelector = (): string => '[type="search"]';
+  getEditButtonSelector = (): string => 'a:contains("Edit")';
+  getDeleteButtonSelector = (): string => 'a:contains("Delete")';
+  getAddCompanyUserRoleButtonSelector = (): string => 'a:contains("Add company user role")';
```

Comment:
the preferred way of working with selectors is to add -> use 'data-qa' attributes. Please check 

https://spryker.atlassian.net/wiki/spaces/CCS/pages/4034854963/What+DOM+selectors+to+use+in+Cypress

In any case, avoid using text in your selectors. In this example, you can use something like

- a contains in href 'create-company-role'
- a contains in href 'edit-company-role' 
- form with name 'delete_form' > button type = submit